cmd/wazero/wazero.go

```diff
@@ -72,6 +72,10 @@ func doCompile(args []string, stdErr io.Writer) int {
 	var help bool
 	flags.BoolVar(&help, "h", false, "Prints usage.")
 
+	var count int
+	flags.IntVar(&count, "count", 1,
```

Comment:
Is there a way to have a flag undocumented? I feel this isn't worth distracting users on -h with. wdyt?

Reply:
I don't think so. 😞

Reply:
We can customize the whole usage message via [`flag.FlatSet.Usage`](https://pkg.go.dev/flag#FlagSet), but I don't know of a way to hide a specific flag.

I think what we could do is expose the option only if the `wazero` version is `"dev"`? So it exists when compiling locally from sources, but not on the release builds.

Reply:
let's do dev. In fact I think you can also consider if the earlier profiler based stuff would make sense under that, too. I think it probably is, as typical CLI users won't use these.